src/newrelic/processor.go

```diff
@@ -72,7 +72,8 @@ const (
 	HarvestErrorEvents  HarvestType = (1 << 6)
 	HarvestSpanEvents   HarvestType = (1 << 7)
 	HarvestLogEvents    HarvestType = (1 << 8)
-	HarvestDefaultData  HarvestType = HarvestMetrics | HarvestErrors | HarvestSlowSQLs | HarvestTxnTraces
+	HarvestPhpPackages  HarvestType = (1 << 9)
+	HarvestDefaultData  HarvestType = HarvestMetrics | HarvestErrors | HarvestSlowSQLs | HarvestTxnTraces | HarvestPhpPackages
 	HarvestAll          HarvestType = HarvestDefaultData | HarvestTxnEvents | HarvestCustomEvents | HarvestErrorEvents | HarvestSpanEvents | HarvestLogEvents
```

Comment:
```suggestion
	HarvestAll          HarvestType = HarvestDefaultData | HarvestTxnEvents | HarvestCustomEvents | HarvestErrorEvents | HarvestSpanEvents | HarvestLogEvents | HarvestPhpPackages
```

Reply:
Since `HarvestPhpPackages` is already in `HarvestDefaultData` I didn't think I needed to add it again in `HarvestAll` which includes `HarvestDefaultData`.